tendrl/commons/objects/cluster_alert/__init__.py

```diff
@@ -0,0 +1,55 @@
+from tendrl.commons.objects.alert import Alert
+
+from tendrl.commons import objects
+
+
+class ClusterAlert(Alert, objects.BaseObject):
+    def __init__(
+        self,
+        alert_id=None,
+        node_id=None,
+        time_stamp=None,
+        resource=None,
+        current_value=None,
+        tags=None,
+        alert_type=None,
+        severity=None,
+        significance=None,
+        ackedby=None,
+        acked=None,
+        ack_comment=[],
+        acked_at=None,
+        pid=None,
+        source=None,
+        delivered=None,
+        *args,
+        **kwargs
+    ):
+        super(ClusterAlert, self).__init__(
```

Comment:
@shtripat ClusterAlert object is inherited from Alert object so initialization is happening using Alert object constructor so i am passing values from here.